cmd/qvain-backend/dataset_api.go

```diff
@@ -119,7 +117,7 @@ func (api *DatasetApi) ListDatasets(w http.ResponseWriter, r *http.Request, user
 
 // Dataset handles requests for a dataset by UUID. It dispatches to request method specific handlers.
 func (api *DatasetApi) Dataset(w http.ResponseWriter, r *http.Request, user *models.User, id uuid.UUID) {
-	//api.logger.Debug().Str("head", "").Str("path", r.URL.Path).Msg("dataset")
+	api.logger.Debug().Str("head", "").Str("path", r.URL.Path).Msg("dataset")
```

Comment:
why commented in?

Reply:
It was commented for some reason, don't know why. but thought it would be helpful to log the URL path of the request coming in for dataset operation.so did uncomment